### PR TITLE
add --public-path option to the command

### PR DESCRIPTION
**Pull request by ggrebert**
add an option in the cli to configure the public path for the UI in dev mode

## What was changed

add `--public-path` option to the command

## Why?

 * creating a quarkus extension for temporal
 * add an easy way to proxify the temporal devservice (`/q/temporal`)

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by cretz** [comment]
This LGTM. I think you'll have to sign the CLA before we can merge.

**Comment by ggrebert** [comment]
> This LGTM. I think you'll have to sign the CLA before we can merge.

done

**Review by cretz — APPROVED** [review]
Thanks! Approved and will wait for @josh-berry's approval. I may also confirm internally with our frontend team that there are no gotchas here with setting this value.

File: temporalcli/commandsmd/commands.md
Comment (cretz) [inline temporalcli/commandsmd/commands.md:1314]:
```suggestion
* `--ui-public-path` (string) -
```

Actually, I'm sorry to just now notice this, but can you add the `ui-` prefix to the arg?

File: temporalcli/commandsmd/commands.md
Comment (josh-berry) [inline temporalcli/commandsmd/commands.md:1316]:
```suggestion
  Default is `/`.
```

Agree with Chad's request to add the `ui-` prefix to this option; we'd like to be consistent with the other UI options.

Otherwise this looks good to me!

File: temporalcli/commands.gen.go
Comment (cretz) [inline temporalcli/commands.gen.go:1318]: @ggrebert - you have to regen from command file since your last change: `go run ./temporalcli/internal/cmd/gen-commands`

File: temporalcli/commands.server.go
Comment (josh-berry) [inline temporalcli/commands.server.go:27]: Looks like an unintentional whitespace change; can you revert this? Otherwise LGTM, thanks!

**Review by josh-berry — APPROVED** [review]

**Comment by cretz** [comment]
Thanks!

**Comment by ggrebert** [comment]
It’s up to me to thank you!

Is it possible to say us when this PR will be available in the `temporalio/auto-setup` Docker image ?

**Pull request merged**
